G1ANT.Addon.Selenium/Variables/TimeoutSeleniumVariable.cs

```diff
@@ -0,0 +1,28 @@
+﻿using G1ANT.Language;
+using System;
+
+namespace G1ANT.Addon.Selenium.Variables
+{
+    [Variable(
+        Name = "timeoutselenium",
+        Tooltip = "Determines the timeout value (in ms) for several `selenium.` commands; the default value is 20000 (20 seconds)")]
```

Comment:
sadly default value is 10 seconds
namespace G1ANT.Addon.Selenium
{
    public static class SeleniumSettings
    {
        public static int SeleniumTimeout = 10000;
    }
}

Reply:
Changed. Good catch!